src/styles/mixins.module.scss

```diff
@@ -34,6 +34,11 @@
   color: #231f20;
 }
 
+@mixin openSansRegular {
+  font-family: 'OpenSans Extra Regular';
```

Comment:
We already have a mixin that uses openSans, im pretty sure its just called openSans, can you use that one? Also there is not font that is called Open Sans Extra Regualr, so Im pretty sure this mixin won't work properly.